Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml

```diff
@@ -36,33 +36,25 @@
     <!-- XHED: Submission consent disclaimer sub headline -->
     <string name="srs_submission_consent_disclaimer_subheadline_text">Bevor Sie andere warnen können, ist Ihr Einverständnis erforderlich.</string>
     <!-- XTXT: Submission consent section warn consent -->
-    <string name="srs_section_warn_consent_text">Sie bestätigen hiermit, dass Sie persönlich durch einen Selbsttest oder von einer Teststelle positiv getestet wurden. Sie können hier nicht in Vertretung von Familienmitgliedern warnen.</string>
+    <string name="srs_section_warn_consent_text">Wenn Sie positiv getestet wurden, können Sie Ihre Mitmenschen über die App warnen.\nDie Warnung nach einem positiven Selbsttest oder mit einem Testergebnis, das nicht in der App vorliegt, erfordert die Prüfung der Echtheit der App.</string>
```

Comment:
```suggestion
    <string name="srs_section_warn_consent_text">Wenn Sie positiv getestet wurden, können Sie Ihre Mitmenschen über die App warnen.\nDie Warnung nach einem positiven Selbsttest oder mit einem Testergebnis, das nicht in der App vorliegt, erfordert die automatische Prüfung der Echtheit der App.</string>
```

I'd suggest to add "automatic" here, to mane clear that there is no manual check involved.

Reply:
Hello @Ein-Tim we will consider your comment for the next release candidate

Reply:
@Ein-Tim I will check with the other colleagues who elaborated on this text whether "automatic" should be added.